.github/labeler.yml

```diff
@@ -0,0 +1,50 @@
+# This file is a map of labels to repo glob paths. See https://github.com/marketplace/actions/labeler
+# for more details.
+
+"area: dds":
+  - any: ["/packages/dds/**", "!/packages/dds/sequence/**"]
+
+"area: dds: sharedstring":
+  - /packages/dds/sequence/**
+
+"area: driver":
+  - /packages/drivers/**
+
+"area: examples":
+  - /examples/**
+
+"area: framework":
+  - /packages/framework/**
+
+"area: host":
+  - /packages/hosts/**
+
+"area: loader":
+  - /packages/hosts/**
+
+# Add "area: repo" label to any root or .github changes
+"area: repo":
+  - any: ["./*", "./.github/**", "!./BREAKING.md"]
+
+"area: runtime":
+  - /packages/runtime/**
+
+"area: server":
+  - /server/**
+
+"area: tools":
+  - /tools/**
+
+"area: website":
+  - any: ["/docs/**", "!/docs/content/**"]
+
+# Any change to BREAKING.md or to any api-report path in the repo
+"breaking change":
+  - ./BREAKING.md
+  - api-report/*
```

Comment:
will this trigger for any api-report folder? even the nested ones?

Reply:
That's my understanding from the docs. I'll update if it doesn't work as expected.